library/src/main/java/com/getstream/sdk/chat/rest/Message.java

```diff
@@ -302,6 +302,7 @@ public Message copy() {
         clone.parentId = parentId;
         clone.command = command;
         clone.commandInfo = commandInfo;
+        clone.isThreadParent = isThreadParent;
```

Comment:
@adrian09h the whole `isThreadParent`, `isThreadParent()` and `setThreadParent` does not make any sense. You can tell a message is the parent of a thread because it is returned by the API with `parent_id` set to `null`. All this extra fields and setters are there to create bugs